var/spack/repos/builtin/packages/xrootd/package.py

```diff
@@ -50,3 +55,6 @@ def install(self, spec, prefix):
             cmake(source_directory, *options)
             make()
             make("install")
+
+    def url_for_version(self, version):
```

Comment:
You're completely right, I thought the `v ` in `v4.3.0` could cause problems during the fetch stage.
I'm deleting it.